src/solidity-frontend/solidity_convert.cpp

```diff
@@ -59,6 +59,77 @@ bool solidity_convertert::convert()
   // We need to convert Solidity AST nodes to the equivalent symbols and add them to the context
   nlohmann::json &nodes = src_ast_json["nodes"];
 
+  // Since the interface itself does not create any vulnerabilities
+  // consider removing it and its inheritance information from the contract
```

Comment:
I agree that the interface would not affect the verification result. However, deleting nodes from the source JSON may not be a good idea. 

May I ask if your approach could support this example?
```solidity
pragma solidity >=0.5.0;

interface caculator {
    enum E {
        x,
        y
    }
}

contract Test {
    function caculator() external pure{
        assert(caculator.E.x <= caculator.E.y);
    }
    function test() external pure{
        assert(caculator.E.x > caculator.E.y);
    }
}
```